### PR TITLE
Add first draft for flux

### PR DESCRIPTION
**Pull request by jan-janssen**
Thanks to @mgt16-LANL

### CODE REVIEW

File: pysqa/wrapper/flux.py [hunk 1]
```diff
@@ -19,7 +19,12 @@ def get_queue_status_command(self):
...
         result = subprocess.run(
-            ['flux', 'job', 'id', queue_submit_output.splitlines()[-1].rstrip().lstrip().split()[-1]],
-            stdout=subprocess.PIPE
+            [
+                "flux",
+                "job",
+                "id",
+                queue_submit_output.splitlines()[-1].rstrip().lstrip().split()[-1],
+            ],
+            stdout=subprocess.PIPE,
```
Comment (jan-janssen) [inline pysqa/wrapper/flux.py]: Maybe there is an easier way to convert the job ID to an integer ID https://github.com/flux-framework/flux-core/issues/4491

Comment (grondo) [reply pysqa/wrapper/flux.py]: Yes, if you can `import flux` then `flux.job.JobID(string)` should convert a Flux jobid in any format to a `JobID` which is a subclass of `int`. (So you could call `int()` on it)

Comment (jan-janssen) [reply pysqa/wrapper/flux.py]: @grondo Somehow I get an `AttributeError: module 'flux' has no attribute 'job'` error

Comment (jan-janssen) [reply pysqa/wrapper/flux.py]: I was able to fix it by importing `from flux.job import JobID`

**Pull request merged**